src/js/edu-benefits/education-wizard.js

```diff
@@ -2,7 +2,7 @@ function toggleClass(element, className) {
   element.classList.toggle(className);
 }
 function openState(element) {
-  element.setAttribute('hidden', false);
+  element.removeAttribute('hidden');
```

Comment:
This file has been deleted in master, so you may get a conflict.

Reply:
Ah gotcha, I heard that mentioned. I'm gonna revert this then and do an ugly override on `[hidden=false` instead. That's probably better anyway.